docs/Gemfile

```diff
@@ -1,2 +1,2 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
Is there a reason these switched to `http` instead of `https`?

Reply:
I was unable to get my bundler to install Gems locally with it being https, not sure what changed from before when I ran it locally. I will ask Mike since he knows ruby stuff.